### ISSUE TITLE
Hen-and-egg problem with adminKeys?

### ISSUE THREAD

**Issue by ojkastl** [cryptpad/helm#14]
https://github.com/cryptpad/helm/blob/c6f73b4d10d57f0fa903debc4d7f8d9ab651bc21/charts/cryptpad/values.yaml#L83

https://docs.cryptpad.org/en/admin_guide/installation.html#configure-cryptpad

```
Once CryptPad is installed, create an account via the Register button on the home page. To make this account an instance administrator:

    Copy their public key found in User Menu (avatar at the top right) > Settings > Account > Public Signing Key
```

Do I understand this correctly, that I would first install this chart without the adminKeys populated? And then upgrade the chart after I modified the chart's values.yaml to include the `adminKeys`?

**Comment by gsautner** [comment]
Yes, we don't have yet something to create an initial user account and put as admin this during first run/install.
